Index: lzma-4.65/C/Archive/7z/makefile.gcc
===================================================================
--- lzma-4.65.orig/C/Archive/7z/makefile.gcc
+++ lzma-4.65/C/Archive/7z/makefile.gcc
@@ -2,7 +2,7 @@ PROG = 7zDec
 CXX = g++
 LIB =
 RM = rm -f
-CFLAGS = -c -O2 -Wall
+CFLAGS = -c -O2 -Wall -fPIC
 
 OBJS = 7zAlloc.o 7zBuf.o 7zBuf2.o 7zCrc.o 7zDecode.o 7zExtract.o 7zHeader.o 7zIn.o 7zItem.o 7zMain.o LzmaDec.o Bra86.o Bcj2.o 7zFile.o 7zStream.o
 
Index: lzma-4.65/C/LzmaUtil/makefile.gcc
===================================================================
--- lzma-4.65.orig/C/LzmaUtil/makefile.gcc
+++ lzma-4.65/C/LzmaUtil/makefile.gcc
@@ -2,7 +2,7 @@ PROG = lzma
 CC = gcc
 LIB = liblzma.a
 RM = rm -f
-CFLAGS = -c -O2 -Wall
+CFLAGS = -c -O2 -Wall -fPIC
 AR = ar
 RANLIB = ranlib
 
Index: lzma-4.65/CPP/7zip/Compress/LZMA_Alone/makefile.gcc
===================================================================
--- lzma-4.65.orig/CPP/7zip/Compress/LZMA_Alone/makefile.gcc
+++ lzma-4.65/CPP/7zip/Compress/LZMA_Alone/makefile.gcc
@@ -1,7 +1,7 @@
 PROG = lzma_alone
 PROG2 = lzma
-CXX = g++ -O2 -Wall
-CXX_C = gcc -O2 -Wall
+CXX = g++ -O2 -Wall -fPIC
+CXX_C = gcc -O2 -Wall -fPIC
 LIB = -lm
 RM = rm -f
 CFLAGS = -c -I ../../../ -D_FILE_OFFSET_BITS=64 -DPACKAGE_VERSION="\"4.32.0beta3\""
